src/main/java/org/alliancegenome/mati/entity/SubdomainEntity.java

```diff
@@ -1,19 +1,19 @@
 package org.alliancegenome.mati.entity;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.SequenceGenerator;
-import javax.persistence.Table;
-import javax.validation.constraints.NotEmpty;
-
 import io.quarkus.hibernate.orm.panache.PanacheEntityBase;
-import lombok.Data;
+import lombok.*;
+import org.hibernate.Hibernate;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import java.util.Objects;
 
 @Entity(name = "Subdomain")
 @Table(name = "subdomain")
-@Data
```

Comment:
Why the change from @Data? you can add the `@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)` annotation that will take care of this.

Reply:
I followed intelliJ suggestion and replacement: remove @Data for the entity to avoid problems in some cases

Reply:
![Screenshot_20221028_101902](https://user-images.githubusercontent.com/143268/198552460-19da281c-5f53-422c-a840-96b5f9a514bf.png)